plugins/src/ios/OptimobileSDKPlugin.swift

```diff
@@ -0,0 +1,9 @@
+import OptimoveSDK
+@objc(Optimobile_Cordova) class OptimobileSwift : CDVPlugin {
+    @objc(initialize:)
+    func initialize(command: CDVInvokedUrlCommand) {
+        let args = command.arguments
+        let config = OptimoveConfigBuilder(optimoveCredentials: args?[0] as? String, optimobileCredentials: args?[1] as? String).enableInAppMessaging(inAppConsentStrategy: .explicitByUser).build()
```

Comment:
We shouldn't enable In-App messaging automatically, not all customers will take that feature.

Reply:
we added a parameter inAppConsentStrategy to determine if the Customer want to add In app messages feature

Reply:
This should likely come from the configuration JSON file instead of an argument, but we can handle it in the follow-up.